doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/data-sets-authoring-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='data-sets-authoring-con_{context}']
+= Data sets authoring
```

Comment:
Per doc style guidelines and for header consistency, please change to:

"Authoring data sets"

Reply:
But it is not a procedure. Can I still use "ing" format?